From 0323ecb758dbf0662c09c6fb90e7e0413c043087 Mon Sep 17 00:00:00 2001
From: Franck LENORMAND <franck.lenormand@nxp.com>
Date: Wed, 25 Jul 2018 17:17:58 +0200
Subject: [PATCH 4358/5242] MLK-19055: driver: crypto: caam: Add build of
 driver on ARCH_MXC_ARM64 config

commit  f395a6a93b27eb5019167464180d7dbd0714a68b from
https://source.codeaurora.org/external/imx/linux-imx.git

The driver is currently built because ARCH_LAYERSCAPE is defined however
this config should not be set for other platforms such as iMX8 family.

This patch add the built of the driver if ARCH_MXC_ARM64 is selected.

Signed-off-by: Franck LENORMAND <franck.lenormand@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/crypto/caam/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/caam/Kconfig b/drivers/crypto/caam/Kconfig
index dac40e9..a7554b4 100644
--- a/drivers/crypto/caam/Kconfig
+++ b/drivers/crypto/caam/Kconfig
@@ -1,6 +1,6 @@
 config CRYPTO_DEV_FSL_CAAM
 	tristate "Freescale CAAM-Multicore driver backend"
-	depends on FSL_SOC || ARCH_MXC || ARCH_LAYERSCAPE
+	depends on FSL_SOC || ARCH_MXC || ARCH_LAYERSCAPE || ARCH_MXC_ARM64
 	select SOC_BUS
 	help
 	  Enables the driver module for Freescale's Cryptographic Accelerator
-- 
1.7.9.5

